src/widgets/search-box/search-box.js

```diff
@@ -113,6 +113,12 @@ const renderer = ({
   }
 };
 
+const disposer = containerNode => () => {
+  const range = document.createRange(); // IE10+
```

Comment:
This one may be more tricky because the input can be provided or built by the widget.

Reply:
Should we replace it by the initial input? I was expecting user should want to remove the widget/input in any cases from the DOM.

Reply:
That's a good question. I would say that since we enhance the checkbox, we should leave it in the state it was provided. But it's a bit tedious to do... No need to worry to much about that though.